tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/Orthogonal.java

```diff
@@ -90,8 +72,9 @@ public Orthogonal(Ops tf, Long seed) {
    * Creates an Orthogonal Initializer
    *
    * @param tf the TensorFlow Ops
-   * @param gain the gain to be applied to the Matrix
-   * @param seed the seed for random number generation
+   * @param gain the gain to be applied to the Matrix. Default is 1.0.
```

Comment:
This default notice should be on the other constructor as this one can't use defaults.

Reply:
OK